modules/sandboxed-containers-deploying-workloads-with-kata-runtime-web.adoc

```diff
@@ -0,0 +1,77 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-web_{context}"]
+= Deploying a workload in a sandboxed container using the web console
+
+{sandboxed-containers-first} installs the kata runtime as a secondary optional runtime on the cluster, and not as the primary runtime.
+
+To deploy a pod-templated workload in a sandboxed container, you must add the kata runtime manually to the workload's YAML file.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+* You have created a `KataConfig` custom resource (CR).
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, expand *Workloads* and select the type of workload you want to create.
+
+. In the workload page, click to create the workload.
```

Comment:
> create the workload
Is this a button on the UI? If yes, add correct formatting.

Reply:
The issue is you can create several different types of workloads, and the button will be **Create <workload_type>** , depending on the one you select. That's why I wrote it this way. Do you have any suggestions to improve this? Because I can't document how to deploy 8 different types of workloads differently, when it's the same procedure each time... :/

Reply:
It wouldn't be **Workload page** because it would be **Deployment page** or **Pod page** etc...

I'm worried the Peer review squad won't want to have so many steps in a single line... :/